content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -697,13 +697,13 @@ mismatch.
 
 ### Pod failure policy {#pod-failure-policy}
 
-{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.26" state="beta" >}}
 
 {{< note >}}
 You can only configure a Pod failure policy for a Job if you have the
 `JobPodFailurePolicy` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
 enabled in your cluster. Additionally, it is recommended
-to enable the `PodDisruptionsCondition` feature gate in order to be able to detect and handle
+to enable the `PodDisruptionConditions` feature gate in order to be able to detect and handle
 Pod disruption conditions in the Pod failure policy (see also:
 [Pod disruption conditions](/docs/concepts/workloads/pods/disruptions#pod-disruption-conditions)). Both feature gates are
 available in Kubernetes v1.25.
```

Comment:
Also revise this line. If you can, pick wording that lets you put `{{< skew currentVersion >}}` in place of a hard coded version number.

Reply:
Applied